src/bpftrace.cpp

```diff
@@ -1790,6 +1795,9 @@ std::string BPFtrace::get_stack(uint64_t stackidpid, bool ustack, StackType stac
       case StackMode::perf:
         stack << "\t" << std::hex << addr << std::dec << " " << sym << std::endl;
         break;
+      case StackMode::raw:
+        // Shouldn't ever get here
```

Comment:
Could we add a `LOG(BUG)` in here as a precaution against future programming errors?